### ISSUE TITLE
A11y_MySQLextensionforAzureDataStudio_Notebooks-Create_NonTextContrast: Color contrast ratio of focus border over selected control is 1.3:1 which is less than 3:1.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio-mysql#169]
"[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### #GitHub Tags:
#A11yMAS; #A11yWCAG2.1; #A11ySev2; #A11yTCS; #DesktopApp; #GH_MySQLextensionforAzureDataStudio_Linux_Mar23; #MySQL extension for Azure Data Studio; #WCAG1.4.11; #Linux; #NonTextContrast; #External; #External:AzureDataStudio; #FirstParty:22678;

### Environment Details:​
Application Name: MySQL extension for Azure Data Studio 
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

### Repro Steps:
1. Launch Azure Data Studio application.
2. TAB to "File" and hit ENTER key. TAB to "New Notebook" and hit ENTER key.
3. TAB to "Cell" dropdown and hit ENTER.
4. Navigate using TAB key and observe that color contrast ratio of focus border over selected control is 1.3:1 which is less than 3:1.

### Actual Result:​
Color contrast ratio of focus border over selected control is 1.3:1 which is less than 3:1.

**Similar issue is also observed in below scenario:
Issue 1: Notebooks - Code cell**
1. Launch Azure Data Studio application.
2. TAB to "File" and hit ENTER key. TAB to "New Notebook" and hit ENTER key.
3. TAB to "Cell" dropdown and hit ENTER. TAB to "Cell Code" and hit ENTER.
4. Navigate using TAB key and observe that color contrast ratio of focus border over selected control is 1.3:1 which is less than 3:1.

### Expected Result:​
Color contrast ratio of focus border over selected control should be greater than or equal to 3:1.

### User Impact:
Low vision users will not be able to see on which control the focus is, and will not be able to select the required list option.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Attachment:
![GithubIssue1](https://user-images.githubusercontent.com/90901519/228740749-c047fffb-37c9-4cc2-98f9-bb0315a3bc47.png)

![Github issue 2](https://user-images.githubusercontent.com/90901519/228742672-dd13eceb-52a5-4fdc-ba78-e688c22c924a.png)

**Comment by nitishsharma25** [comment]
This issue is external to MySQL extension as it is a common UI of Azure Data Studio.

**Comment by PoojaNamde** [comment]
@nitishsharma25

As per the external bug process we have to log external bugs to MySQL extension. Please reactivate all the bugs and share all the details like VSO, area path details to log an external bug

**Comment by PoojaNamde** [comment]
Logged External bug: https://github.com/microsoft/azuredatastudio/issues/22678

As per the [External Bug Process - First Party Dependencies](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDV8MDF8fGM4NTQxMGIxZGFhYjQwMTZmNTIzMDhkYWM5NDgyMWQwfDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2MzgwNDM2MDg4Njg1MjIxNDZ8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxNVFkyT0RjMk5EQTROVFkzT1RzeE5qWTROelkwTURnMU5qYzVPekU1T2pjelpqUTNOV00zTFRnNU5Ea3RORFprT1MxaFlURTVMVGN3WlRobU9EWm1ZVGszT0Y4M1lqbG1ZVFV5TmkweVptRXhMVFF5WmpRdFltVXhOeTFsTXpRME5EbGpabVkyTlRSQWRXNXhMbWRpYkM1emNHRmpaWE09fDA5YTBhOTU1NzM0MDQ5NTM1OTJiMDhkYWM5NDgyMWQwfGZlNmZiOTdlMzBhNTRjNzJhNTRhNTU0Y2RjNDg0NzM3&sdata=ZVRCdGg0WldPVlRVQ2RveFFNV1FJUEc0bHc1a2QvWTExbkVLS3V5emQzYz0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-ramakatare%40microsoft.com) assigning the bug to the Product Owner. Please assign this bug back to the tester once the "External" issue has been resolved and changes are pushed to the environment that will be available for bug verification.

**Comment by PoojaNamde** [comment]
#Regressed:05-25-23; #Closed;

As per the SME input, closing this bug.
